docs/tutorials/login-metamask.md

```diff
@@ -3,7 +3,202 @@ sidebar_position: 3
+
+```bash
+npm install --save @nevermined-io/catalog-providers
+```
+  </TabItem>
+</Tabs>
+
+## Config your network (optional)
+At least that you need to set differents networks for your dapp than [polygon](https://polygon.technology/) this file is not needed, anyway here we show you how should look the config file.
+
+```ts
+import { zeroX } from '@nevermined-io/nevermined-sdk-js/dist/node/utils';
+import { acceptedChainId } from 'config';
+
+const acceptedChainIdHex = zeroX((+acceptedChainId).toString(16));
+const spreeChainId = zeroX((8996).toString(16));
+const polygonLocalnetChainId = zeroX((8997).toString(16));
+export const mumbaiChainId = zeroX((80001).toString(16));
+const mainnetChainId = zeroX((137).toString(16));
+
+const ChainConfig = {
+  development: {
+    chainId: acceptedChainIdHex === spreeChainId ? spreeChainId : polygonLocalnetChainId,
+    chainName: 'Localhost development',
+    nativeCurrency: {
+      name: 'Ethereum',
+      symbol: 'ETH',
+      decimals: 18
+    },
+    rpcUrls: ['http://localhost:8545'],
+    blockExplorerUrls: [''],
+    iconUrls: ['https://s2.coinmarketcap.com/static/img/coins/64x64/3890.png']
+  },
+  mumbai: {
+    chainId: mumbaiChainId,
+    chainName: 'Polygon Testnet Mumbai',
+    nativeCurrency: {
+      name: 'Matic',
+      symbol: 'MATIC',
+      decimals: 18
+    },
+    rpcUrls: [
+      'https://matic-mumbai.chainstacklabs.com',
+      'https://rpc-endpoints.superfluid.dev/mumbai'
+    ],
+    blockExplorerUrls: ['https://mumbai.polygonscan.com/'],
+    iconUrls: ['https://s2.coinmarketcap.com/static/img/coins/64x64/3890.png']
+  },
+  mainnet: {
+    chainId: mainnetChainId,
+    chainName: 'Polygon Mainnet',
+    nativeCurrency: {
+      name: 'Matic',
+      symbol: 'MATIC',
+      decimals: 18
+    },
+    rpcUrls: ['https://polygon-rpc.com'],
+    blockExplorerUrls: ['https://polygonscan.com/'],
+    iconUrls: ['https://s2.coinmarketcap.com/static/img/coins/64x64/3890.png']
+  },
+  returnConfig: (chainIdHex: string) => {
+    if (chainIdHex === spreeChainId || chainIdHex === polygonLocalnetChainId) {
+      return ChainConfig.development;
+    }
+    if (chainIdHex === mumbaiChainId) {
+      return ChainConfig.mumbai;
+    }
+    if (chainIdHex === mainnetChainId) {
+      return ChainConfig.mainnet;
+    }
+    return ChainConfig.development;
+  }
+};
+
+export default ChainConfig;
+```
+
+This config file represent all the networks that your dapp support, and a default one where metamask will ask to switch in case that the wallet is in some not supported network
+
+## Add the Metamask provider
+Now in our app we need to add the Metamask provider as a parent of all the components that will use functionalities from it
+
+```tsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Example from 'examples';
+import { MetaMask } from '@nevermined-io/catalog-providers';
+import chainConfig, { mumbaiChainId } from './chain_config';
```

Comment:
here instead of importing `chainConfig` we should write down the chain config for mumbai